tidy3d/components/geometry.py

```diff
@@ -16,7 +16,11 @@
 from .viz import add_ax_if_none, equal_aspect
 from .viz import PLOT_BUFFER, ARROW_LENGTH_FACTOR, ARROW_WIDTH_FACTOR
 from ..log import Tidy3dKeyError, SetupError, ValidationError
-from ..constants import MICROMETER, LARGE_NUMBER
+from ..constants import MICROMETER, LARGE_NUMBER, RADIAN
+
+# for sampling polygon in slanted polyslab along  z-direction for
+# validating polygon to be non_intersecting.
+_N_SAMPLE = 100
```

Comment:
can we make this variable more specific like `_N_SAMPLE_POLYGON_INTERSECT`

Reply:
done